values.yaml

```
@@ -329,125 +444,168 @@ server:
               component: server
           topologyKey: kubernetes.io/hostname
 
-  # Toleration Settings for server pods
-  # This should be a multi-line string matching the Toleration array
-  # in a PodSpec.
+  # Toleration settings for server pods. This
+  # should be a multi-line string matching the Tolerations
+  # (https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) array in a Pod spec.
   tolerations: ""
 
-  # nodeSelector labels for server pod assignment, formatted as a multi-line string.
-  # ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+  # This value defines `nodeSelector` (https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector)
+  # labels for server pod assignment, formatted as a multi-line string.
+  #
   # Example:
+  #
+  # ```yaml
   # nodeSelector: |
   #   beta.kubernetes.io/arch: amd64
+  # ```
+  #
+  # @type: string
   nodeSelector: null
 
-  # used to assign priority to server pods
-  # ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+  # This value references an existing
+  # Kubernetes `priorityClassName` (https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#pod-priority)
+  # that can be assigned to server pods.
   priorityClassName: ""
 
-  # Extra labels to attach to the server pods.
-  # This should be a regular YAML map.
+  # Extra labels to attach to the server pods. This should be a YAML map.
+  #
   # Example:
-  #   extraLabels:
-  #     labelKey: "label-value"
-  #     otherLabelKey: "another-label-value"
+  #
+  # ```yaml
+  # extraLabels:
+  #   labelKey: label-value
+  #   anotherLabelKey: another-label-value
+  # ```
+  #
+  # @type: map
   extraLabels: null
 
-  # Extra annotations to attach to the server pods.
-  # This should be a multi-line YAML string.
-  # Example:
-  #   annotations: |
-  #     "annotation-key": "annotation-value"
+  # This value defines additional annotations for
+  # server pods. This should be a formatted as a multi-line string.
+  #
+  # ```yaml
+  # annotations: |
+  #   "sample/annotation1": "foo"
+  #   "sample/annotation2": "bar"
+  # ```
+  #
+  # @type: string
   annotations: null
 
+  # Server service properties.
   service:
     # Annotations to apply to the server service.
-    # Example:
-    #   annotations: |
-    #     "annotation-key": "annotation-value"
+    #
+    # ```yaml
+    # annotations: |
+    #   "annotation-key": "annotation-value"
+    # ```
+    #
+    # @type: string
     annotations: null
 
-  # extraEnvVars is a list of extra environment variables to set with the stateful set. These could be
-  # used to include proxy settings required for cloud auto-join feature,
-  # in case kubernetes cluster is behind egress http proxies. Additionally, it could be used to configure
-  # custom consul parameters.
+  # extraEnvironmentVars is a list of extra environment variables to set within the stateful set.
```

```suggestion
  # A list of extra environment variables to set within the stateful set.
```
Ignore if you feel this is unneccessary, just following the pattern of the other changes I see here.